### PR TITLE
VB-3616 Past bookings page - MVP

### PR DESCRIPTION
**Pull request by AledWEvans**
added past bookings

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PublicVisitController.kt [hunk 1]
```diff
@@ -18,6 +18,7 @@ import uk.gov.justice.digital.hmpps.visitscheduler.service.VisitService
 
 const val GET_BOOKED_FUTURE_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/booked/visits/future"
 const val GET_CANCELLED_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/cancelled/visits"
+const val GET_BOOKED_PAST_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/past/visits"
```
Comment (adaviesMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PublicVisitController.kt:21]: Can this URL be:
/public/booker/{bookerReference}/booked/visits/past similar to the future URL?

Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PublicVisitController.kt]: should the endpoint be similar to - /public/booker/{bookerReference}/booked/visits/future i.e. /public/booker/{bookerReference}/booked/visits/past?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PublicVisitController.kt]: have changed as discussed.

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by adaviesMOJ — APPROVED** [review]

**Review by adaviesMOJ — APPROVED** [review]

**Pull request merged**